server/src/test/java/org/elasticsearch/index/mapper/CompletionFieldMapperTests.java

```diff
@@ -38,6 +38,7 @@
 import org.elasticsearch.common.xcontent.XContentBuilder;
 import org.elasticsearch.common.xcontent.XContentFactory;
 import org.elasticsearch.common.xcontent.XContentType;
+import org.elasticsearch.common.xcontent.XContentParser.Token;
```

Comment:
Sorry I missed that we forbid unused import in our build so this needs to be removed.

Reply:
already fixed :) thanks